tests/bgp/test_bgp_gr_helper.py

```diff
@@ -17,18 +17,17 @@ def test_bgp_gr_helper_routes_perserved(duthosts, rand_one_dut_hostname, nbrhost
     """
     duthost = duthosts[rand_one_dut_hostname]
 
+    if not duthost.check_bgp_default_route():
+        pytest.skip("there is no nexthop for bgp default route")
+
```

Comment:
As per the current flow in 'setup_bgp_graceful_restart', we reach this point only when its a backend topology or if there is a bgp default route present on other topologies and this check will be true only for 'backend'
But I am not clear on why then lines 26-27 and lines 30-31 are removed

Reply:
Checks in lines 26-27 and lines 30-31 actually could not fail on the storage backend topology, the check could pass because there is a static default route. So I use `check_bgp_default_route` to check the default route that is received/learned from a BGP neighbor.